src/opentelemetry/launcher/configuration.py

```diff
@@ -30,18 +30,18 @@
 from environs import Env
 from grpc import ssl_channel_credentials
 
-from opentelemetry.baggage.propagation import BaggagePropagator
+from opentelemetry.baggage.propagation import W3CBaggagePropagator
 from opentelemetry.instrumentation.distro import BaseDistro
 from opentelemetry.launcher.tracer import LightstepOTLPSpanExporter
-from opentelemetry.launcher.version import __version__
-from opentelemetry.propagators import set_global_textmap
+from opentelemetry.propagate import set_global_textmap
 from opentelemetry.propagators.b3 import B3Format
 from opentelemetry.propagators.composite import CompositeHTTPPropagator
 from opentelemetry.sdk.trace import Resource, TracerProvider
 from opentelemetry.sdk.trace.export import (
-    BatchExportSpanProcessor,
+    BatchSpanProcessor,
     ConsoleSpanExporter,
 )
+from opentelemetry.sdk.version import __version__
```

Comment:
Was this intentional? I'd expect the launcher version to be used here.

Reply:
Yes, it is intentional. What happens is that the code now checks if an attribute has been set already and does not override it. The `"telemetry.sdk.version"` attribute in the `Resource` is set [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py#L208). If we don't use the SDK version and use the launcher `__version__` tests will fail, since the default `Resource` will come with this attribute already set to the SDK version number.

Also, shouldn't `"telemetry.sdk.version"` actually match the SDK version always?

Reply:
if this is the case, the version is already set, is this import still needed? i'm not sure i understand why this is needed here.

Reply:
Right, I have removed it from `configuration`.